### PR TITLE
Adjust documentation and example of IsPrimitiveCharacter

### LINKED ISSUE #3779: Documentation of IsPrimitiveCharacter is unclear
On 5th of December Benjamin Sambale reported the following:

I was a bit confused that IsPrimitiveCharacter does not work with A5. Maybe one should add to the documentation that this command is only implemented for solvable groups. Moreover, the example in that documentation refers to IsPrimitive instead of IsPrimitiveCharacter.

### PR DESCRIPTION
**Pull request by DominikBernhardt**
Fixes #3779 .

### CODE REVIEW

**Comment by wucas** [comment]
Something went wrong with travis, `wget` couldn't resolve `www.gap-system.org`. I restarted it.

**Comment by DominikBernhardt** [comment]
This means that you implicitly define `G` in the sentence. You just want to give something a name.  Afaik it's standard to write it that way. The sentence is equivalent to :
Let G be a finite group and chi a character of G...

**Comment by DominikBernhardt** [comment]
@fingolfin It's a stylistical question I don't really have a preference on. I get your point, so I changed the PR accordingly.

File: lib/ctblmono.gd [hunk 1]
```diff
@@ -298,12 +298,13 @@ DeclareGlobalFunction( "TestHomogeneous" );
 ##  <Description>
 ##  For a character <A>chi</A> of the group <M>G</M>, say,
```
[the remark this answers is not in the document]
Comment (wucas) [reply lib/ctblmono.gd]: Yes I think that's clearer. Thank you!

**Comment by wilfwilson** [comment]
(I started writing my review before Max merged, in case anyone was wondered whether I had reviewed an already-merged PR 😉)

**Pull request merged**